src/core/models/quarters.ts

```diff
@@ -0,0 +1,51 @@
+import { Schema, model } from "mongoose";
+
+const QuarterSchema = new Schema({
+    
+    currentYear: { 
+        type: Number,
+        //i was thinking of using date for this field so this can be automated rather than us putting it in manually.
+        //type: Date,
+        min: 2024,
+    },
+
+    currentQuarter: {
+        type: Number,
+        min: 1,
+        max: 4,
+        required: true
+    },
+
+    starting_capital: {
+        type: Float64Array,
+        min: 0
+    },
+
+    ending_capital: {
+        type: Float64Array,
+        min: 0
+    },
+
+    roi: {
+        type: Float64Array,
+        required: true,
+        min: 0
+    },
+
+    commission: {
+        type: Boolean,
+        default: true
+    },
+
+    account: {
+        type: Schema.Types.ObjectId,
+        ref: 'Account'
+    }
+
```

Comment:
No link to the user table?

Reply:
really is that need. i thought connecting to the accounts table was it.

if we have 10 client we will have 10 records in out accounts table 
or is the user reference an extra layer to be sure we are assigning the information to the right user account?

Reply:
Okay, what you've done is fine then